### ISSUE TITLE
Duplicate keys not allowed

### ISSUE THREAD

**Issue by eserte** [niczero/mojo-jsonxs#2]
I see a new failure on my smokers:
```
# Testing Mojo::JSON_XS 1.003, Perl 5.026000, /opt/perl-5.26.0/bin/perl
test/00-load.t ... ok
Duplicate keys not allowed, at character offset 12 (before "foo": 2}") at /home/cpansand/.cpan/build/2017072918/Mojo-JSON_XS-1.003-0/blib/lib/Mojo/JSON_XS.pm line 18.
# Tests were run but no plan was declared and done_testing() was not seen.
# Looks like your test exited with 255 just after 82.
test/10-json.t ... 
Dubious, test returned 255 (wstat 65280, 0xff00)
All 82 subtests passed 
```
This is probably caused by this change in Cpanel::JSON::XS:
https://metacpan.org/source/RURBAN/Cpanel-JSON-XS-3.0237/Changes#L15

**Comment by karenetheridge** [comment]
This failure continues to exist. The test should be changed to avoid the problematic data.

**Comment by karenetheridge** [comment]
fixed in #3.

**Comment by niczero** [comment]
These tests are passing for v1.01
